test/README.md

```diff
@@ -0,0 +1,53 @@
+# Tests
+
+## Structures
+
+There are 2 suites of tests here: [`e2e`](./e2e) and [`osd`](./osd). 
+
+* [`e2e`](./e2e)
+  
+  The tests in this directory are executed using the operator-sdk's test command against an OCP environment, and will be exectued everytime when a PR is opened.
+ 
+* [`osd`](./osd)
+
+  The tests in this directory are executed from inside of a container, as part of the [OSD Addon testing flow](https://github.com/openshift/osde2e/blob/master/docs/Addons.md), against an OSD environment. 
+  Currently these tests are executed periodically (every 4 hours).
```

Comment:
The tests in this directory will be built into a container for testing via the osde2e add on flow, but can they not be executed outside of that?

Is there anything osd specific in these tests? Would this not be considered the functional test suite rather than anything sod specific.